kentik_api_library/tests/library/test_imports.py

```diff
@@ -1,20 +1,16 @@
 # pylint: skip-file
-import sys
 import importlib
+import sys
 
 
 def test_import(library=""):
-
-    library_found = False
-
+    error = None
     try:
-        imported_library = importlib.import_module(library)
-    except ImportError:
-        pass
-    else:
-        library_found = True
+        _ = importlib.import_module(library)
+    except ImportError as err:
+        error = err
 
-    assert library_found, "Can't import '" + library + "'"
+    assert error is None, f'Can\'t import "{library}": {error}'
```

Comment:
Let's keep double-quotes at the top level, like this:
```
assert error is None, f"Can't import '{library}': {error}"
```

It is more readable and `black` has no problem with this (at least in my quick test).